python/src/pymain.cc

```diff
@@ -1,19 +1,9 @@
 #include <boost/python.hpp>
-#include "options.cc"
-#include "Algebra.h"
-#include "Catalog.h"
-#include "Greyscale.h"
-#include "HeuristicOptimizer.h"
-#include "Visitor.h"
-#include "Coordinator.h"
-#include "extension.h"
-#include "reference.h"
-#include "options.h"
-#include "LightField.h"
+#include "PythonLightField.h"
```

Comment:
Rename `pymain.cc` to `Module.cc`

Reply:
Remove superfluous `this` indirections.
Remove typedefs.
Fix spelling of `theth` to `theta`
Remove `SetupLocalEnvironment` or add TODO
Remove pragmas